src/patient-lab-dashboard/table/report-table/report-table.component.tsx

```diff
@@ -75,6 +97,25 @@ const ReportTable = (props: ReportTableProps) => {
     return selfPatient
   }
 
+  useEffect(() => {
+    if (testResults) {
+      setLabTestResults({...testResults})
+      const labOrder = testResults?.data?.results[0]
+      labOrder &&
+        getTestsInLabOrder(labOrder)?.map(sample => {
+          getTestsInLabOrder(sample).map((tests: LabTest) => {
```

Comment:
map creates a new array and  it is advisable not to use map unless we need to create a new array